### DISCUSSION TITLE
xstate browser compatibility

### DISCUSSION THREAD

**Discussion by joseangel-mm** [statelyai/xstate#3851] [Q&A]
Hi all, is there any place where I can find the browsers compatible (and their version)? I could find something in the official documentation.

**Comment by davidkpiano** [comment]
XState is compatible with all modern browsers. Are there any specific browsers you are wondering about?

**Reply by joseangel-mm** [reply]
@davidkpiano Well, actually I'd like to know which browser versions are compatible in order to know if it fits well with our project where we give some support for previous browser versions. I meant, for example in Chrome we give support from 102 version to the current one, so I'd like to know if XState is able to give compatibility with old (not too hehe) versions.

**Reply by davidkpiano** [reply]
If it helps, the TypeScript target is `es5` for XState v4.

**Reply by joseangel-mm** [reply]
Sure! Thanks, @davidkpiano.
